tests/test_rerere.py

```diff
@@ -356,9 +356,9 @@ def flip_last_two_commits(repo: Repository, ed: Editor) -> None:
         )
 
 
-def dedent(text: str) -> bytes:
-    return textwrap.dedent(text).encode()
+def dedent_as_bytes(text: str) -> bytes:
+    return dedent(text).encode()
```

Comment:
This is just a renaming, to avoid aliasing with the regular dedent function, so that calls to `dedent` will do the regular thing. But I think we shouldn't have this function either: It exists to compare `bytes` to `str`, and it chooses to compare them as bytes. But as I hint at in the commit message, test failures are more readable with string comparisons. Which speaks for doing the opposite.

Reply:
Did the opposite. State before: e22fad5